frontend/packages/console-plugin-sdk/src/typings/perspective.ts

```diff
@@ -0,0 +1,24 @@
+import { Extension } from '.';
+
+namespace ExtensionProperties {
+  export interface Perspective {
+    /* The perspective identifier. */
+    id: string;
+    /* The perspective display name. */
+    name: string;
+    /* The perspective display icon. */
+    icon: React.ReactNode;
+    /* The perspective landing page URL. */
+    landingPageUrl: string;
```

Comment:
nit: with current console naming conventions, this would be `URL`

```suggestion
    landingPageURL: string;
```

Reply:
Updated to `landingPageURL`